src/commands/cronjob/update.tsx

```diff
@@ -0,0 +1,133 @@
+import { ExecRenderBaseCommand } from "../../lib/basecommands/ExecRenderBaseCommand.js";
+import { Args, Flags } from "@oclif/core";
+import { ReactNode } from "react";
+import {
+  makeProcessRenderer,
+  processFlags,
+} from "../../rendering/process/process_flags.js";
+import { Success } from "../../rendering/react/components/Success.js";
+import assertSuccess from "../../lib/apiutil/assert_success.js";
+import type { MittwaldAPIV2Client } from "@mittwald/api-client";
+import { cronjobFlagDefinitions } from "../../lib/resources/cronjob/flags.js";
+import { buildCronjobDestination } from "../../lib/resources/cronjob/destination.js";
+
+type UpdateResult = void;
+type CronjobUpdateData = Parameters<
+  MittwaldAPIV2Client["cronjob"]["updateCronjob"]
+>[0]["data"];
+
+export default class Update extends ExecRenderBaseCommand<
+  typeof Update,
+  UpdateResult
+> {
+  static description = "Update an existing cron job";
+  static args = {
+    "cronjob-id": Args.string({
+      description: "ID of the cron job to be updated.",
+      required: true,
+    }),
+  };
+  static flags = {
+    ...processFlags,
+    description: cronjobFlagDefinitions.description(),
+    interval: cronjobFlagDefinitions.interval(),
+    email: cronjobFlagDefinitions.email(),
+    timeout: cronjobFlagDefinitions.timeout(),
+    url: cronjobFlagDefinitions.url({
+      exclusive: ["command"],
+    }),
+    command: cronjobFlagDefinitions.command({
+      exclusive: ["url"],
+      dependsOn: ["interpreter"],
+    }),
+    interpreter: cronjobFlagDefinitions.interpreter({
+      dependsOn: ["command"],
+    }),
+    enable: Flags.boolean({
+      exclusive: ["disable"],
+      summary: "Enable the cron job.",
+      description:
+        "Set the status of the cron job to inactive. Automatic execution will be disabled.",
+    }),
+    disable: Flags.boolean({
+      exclusive: ["enable"],
+      summary: "Disable the cron job.",
+      description:
+        "Set the status of the cron job to active. Automatic execution will be enabled.",
+    }),
+  };
+
+  protected async exec(): Promise<void> {
+    const process = makeProcessRenderer(this.flags, "Updating cron job");
+    const { "cronjob-id": cronjobId } = this.args;
+    const currentCronjob = await this.apiClient.cronjob.getCronjob({
+      cronjobId,
+    });
+    assertSuccess(currentCronjob);
```

Comment:
I can't see `currentCronjob` being used anywhere. Is this a leftover from a previous iteration?

Reply:
leftover from a different active/inactive implementation.
gone now